docs/src/folderstructure.md

```diff
@@ -0,0 +1,54 @@
+
+For example, `/experiments/LSM/ozark/` contains:
+- ozark_domain.jl: Describes the soil domain (depth, number of layer), and the canopy (number and height of stems and leaves).
+- ozark_met_drivers_FLUXNET.jl: This files load meteorological input data from the ozark FLUXNET file, and does additional things such as spline interpolation of these drivers.
+- ozark_parameters.jl: In this file, parameters values are defined. 
+- ozark_simulation.jl: In this file, initial and final time are set, as well as time resolution and time stepper algorithm. 
+- ozark.jl: running this script will include all the above scripts, and run ClimaLSM for the single-site ozark. It will produce output in a text file as well as some figures comparing data and simulation.
+
+## /src folder
+
+The `/src` folder contains the source code of `ClimaLSM` models. It contains 3 folders:
+- SharedUtilities: This is a core folder that defines functions and data structures used across all modules and models types of `ClimaLSM`. For example, `SharedUtilities/models.jl` defines and export the function `make_update_aux` which will be used to create a function which updates the auxiliary parameters, stored in the vector `p`, `SharedUtilities/boundary_conditions.jl` defines functions for setting boundary condition for PDE domains, etc.
+- Standalone: This folder contains standalone models, which are submodels that can be run independently of each other. This is an important aspect of `ClimaLSM` code design: to maximise modularity, sub-models can be run alone, and many different methods of the same sub-model can be defined via Julia multiple-disptach. The standalone folder is independent from the Integrated folder. 
+- Integrated: This folder contains integrated models. It assembles standalone models together, as one would assemble pieces of a puzzle. Thanks to the modularity of `ClimaLSM` design, many configuration of LSM can be assembled in integrated models. The same functions (`update_aux!`, `exp_tendency!`, etc.) can be used for standalone and integrated models, and an can be stepped  in the same way.
+
+As well as 3 files:
+- ClimaLSM.jl: This file is the main Julia module of `ClimaLSM.jl` repository. It contains all functions defined in `/src` in a nested way, for example `ClimaLSM.X`, `ClimaLSM.Soil.X`, 'ClimaLSM.Canopy.X`, etc. When a Julia user install and uses ClimaLSM via `]add ClimaLSM, using ClimaLSM`, they are loading those functions, and are ready to use ClimaLSM codebase. 
+- Parameters.jl: This script describes a single interface and parameter set that all CliMA models use. This ensures all components of the Earth System Model use the same values for all parameters, and that the same parameter calibration tools can be used to calibrate all models, either jointly or independently.
```

Comment:
oh! one other quick fix - these descriptions can now be moved under "SharedUtilities", since you moved them :)